back/engines/free/email_campaigns/spec/mailers/previews/user_blocked_mailer_preview.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class UserBlockedMailerPreview < ActionMailer::Preview
+  def send_user_blocked_email
+    user = User.first
+    user.update(block_start_at: 5.days.ago, block_reason: 'You breached guideline X')
+    UserBlockedMailer.with(user: user).send_user_blocked_email
+  end
+end
```

Comment:
@alexander-cit This doesn't work. It doesn't show on list @ http://localhost:4000/rails/mailers/ I'm clearly missing something important here.

Unfortunately only other preview that is not a campaign mail, `back/engines/free/email_campaigns/spec/mailers/previews/user_confirmation/confirmations_mailer_preview.rb` doesn't work. When I click on `send_confirmation_code` I get the error...```
uninitialized constant UserConfirmation::ConfirmationsMailer
```
So I can't just copy how that is implemented.

Ideas for how to get this working would be greatly appreciated!

Reply:
You can use this code, it works

```ruby
module UserConfirmation
  class ConfirmationsMailerPreview < ActionMailer::Preview
    def send_confirmation_code
      user = User.first
      user.save
      ::ConfirmationsMailer.with(user: user).send_confirmation_code
    end
  end
end
```

Reply:
Also, you need to restart `docker-compose` after changes.

Reply:
Yes! Thank you. Restarting the container did the trick. 🙌